apps/coscrad-frontend/src/components/resources/audio-item/audio-item-detail.full-view.presenter.tsx

```diff
@@ -36,7 +37,12 @@ export const AudioItemDetailFullViewPresenter = ({
     ));
 
     return (
-        <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.audioItem}>
+        <ResourceDetailFullViewPresenter
+            name={name}
+            id={id}
+            type={ResourceType.audioItem}
+            contributions={contributions}
```

Comment:
We should make sure to update fixture data for front-end jest tests now that the contract has changed.

Reply:
done